src/main/scala/com/wavesplatform/matcher/MatcherSettings.scala

```diff
@@ -18,7 +20,10 @@ case class MatcherSettings(enable: Boolean,
                            snapshotsInterval: FiniteDuration,
                            maxOpenOrders: Int,
                            priceAssets: Seq[String],
-                           predefinedPairs: Seq[AssetPair]
+                           predefinedPairs: Seq[AssetPair],
+                           maxTimestampDiff: FiniteDuration,
+                           orderHistoryFile: String,
+                           isMigrateToNewOrderHistoryStorage: Boolean
```

Comment:
It's good that it's calculable, but it still shouldn't be here. It is not a "setting" of any kind.